marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/MintBurn.hs

```diff
@@ -333,109 +310,106 @@ rewind = H.property $ do
       expected = filter isBeforeRollback events
   equalSet expected (MintBurn.fromRows queryResult)
 
+-- TODO Commented because cardano-testnet was out of date and we needed to update cardano-api.
```

Comment:
Maybe add this comment as a note and move it to `marconi-chain-index/test-lib/Helpers.hs` and refers to it here?

Reply:
Good idea!